src/QuickMarcEditor/QuickMarcEditorRows/BytesField/BytesField.js

```diff
@@ -102,6 +103,7 @@ const renderSubField = (name, config) => {
           className={styles[`bytesFieldSubField${config.type}`]}
           hasClearIcon={false}
           data-testid={`fixed-field-${config.type}`}
+          defaultValue={defaultValue}
```

Comment:
`defaultValue` makes the form state dirty instead of pristine, as a result, the `Save&Close` button is enabled. It would be nice to add default values to `initialValues`.

Reply:
This is caused by `Ctrl` field - BE doesn't send this value at all so on UI we add the default value.
For all other fields - if we send empty values 
![image](https://github.com/folio-org/ui-quick-marc/assets/19309423/0c71fcd5-c604-475b-afad-1791e5a5fffb)
BE upon save will auto-fill them with `\`
![image](https://github.com/folio-org/ui-quick-marc/assets/19309423/bb0afee1-f4fd-46c9-8b34-4f89b855c2a3)

Reply:
In edit mode, when we just opened a record the `Save&Close` button is enabled, QA can create a bug for this. Is it possible to add the a value for `Ctrl` to the `initialValues` prop, it should fix this behavior.

Reply:
We're going to remove `Ctrl` and we have a ticket in MOL backlog to do it
I'll ask if we can take this ticket - it's very small and can be done quickly